imod/msw/landuse.py

```diff
@@ -160,12 +165,22 @@ def __init__(
         self.dataset["start_sprinkling_season"] = start_sprinkling_season
         self.dataset["end_sprinkling_season"] = end_sprinkling_season
         self.dataset["interception_option"] = interception_option
-        self.dataset["interception_capacity_per_LAI_Rutter"] = (
-            interception_capacity_per_LAI
-        )
-        self.dataset["interception_capacity_per_LAI_VonHoyningen"] = (
-            interception_capacity_per_LAI
-        )
+        if interception_capacity_per_LAI is not None:
+            self.dataset["interception_capacity_per_LAI_Rutter"] = (
+                interception_capacity_per_LAI
+            )
+        else:
+            self.dataset["interception_capacity_per_LAI_Rutter"] = (
+                interception_capacity_per_LAI_Rutter
+            )
+        if interception_capacity_per_LAI is not None:
+            self.dataset["interception_capacity_per_LAI_VonHoyningen"] = (
+                interception_capacity_per_LAI
+            )
+        else:
+            self.dataset["interception_capacity_per_LAI_VonHoyningen"] = (
+                interception_capacity_per_LAI_VonHoyningen
+            )
```

Comment:
The reason is this:
-in most datasets, the list of keys in the dataset matches the arguments of the` __init__ ` function of the package
-so in `_regrid` we can regrid all the arrays, and then call `__init__` on the regridded package 
-but for this package, the arguments of the` __init__` function did not match the keys of the package dataset because one init parameter( xxx)  was placed in the dataset under 2 names:  xxx_VonHoyningen and xxx_Rutter
-so now I added the option to pass  xxx_VonHoyningen and xxx_Rutter directly to the initializer , so the _regrid function will work just like for any other package.
-I also preserved the original parameter so that user code wil keep working
-I now updated the docstring to inform users that the new init-parameters are for internal use and the user should ignore them

Reply:
Thanks for the explanation, this makes sense.